app/controllers/report_controller.rb

```diff
@@ -5,42 +5,29 @@ class ReportController < CatalogController
   helper ArgoHelper
   copy_blacklight_config_from CatalogController
 
+  ROWS_PER_PAGE = 100
+
   def rsolr_request_error(exception)
     raise exception
   end
 
   def data
-    params[:sord] ||= "asc"
-    rows_per_page = params[:rows] ? params.delete(:rows).to_i : 10
-    params[:per_page] = rows_per_page * [params.delete(:npage).to_i, 1].max
+    params[:per_page] = ROWS_PER_PAGE
+    params[:page] ||= 1
     @report = Report.new(params, current_user:)
 
     respond_to do |format|
       format.json do
         render json: {
           page: params[:page].to_i,
-          records: @report.num_found,
-          total: (@report.num_found / rows_per_page.to_f).ceil,
-          rows: @report.report_data
+          last_page: (@report.num_found / ROWS_PER_PAGE.to_f).ceil,
+          data: @report.report_data
```

Comment:
Adjust what's returned to be what the client-side table library (Tabulator) wants.